tests/utils/test_geometry_comparisons.py

```diff
@@ -221,6 +222,41 @@ def test_compute_relative_rotation_angle(self):
 
         np.testing.assert_allclose(computed_deg, expected_deg, rtol=1e-3, atol=1e-3)
 
+
+    def test_compute_relative_rotation_angle(self) -> None:
```

Comment:
How does this test pass when GTSAM's comparisons are buggy?

Reply:
@ayushbaid So the GTSAM axisAngle *usually* works. Just not for a few edge cases. Over 50,000 image pairs, this can be 100+ completely erroneous values, making our plots look like garbage. Scipy seems to handle these edge cases properly currently. So this unit test here is checking some generic cases, but can’t sample the edge cases. So it passes.